app-near/src/app/_components/survey/SurveyFooter.tsx

```diff
@@ -0,0 +1,25 @@
+const SurveyFooter: React.FC = () => {
+  return (
+    <div className="my-8 flex flex-col gap-8">
+      <div className="flex flex-wrap justify-center gap-8 bg-yellowLight px-12 py-4 md:justify-between">
+        <div className="my-auto flex flex-wrap justify-center gap-8">
+          <img className="h-[61px]" src="/logos/abc.webp" alt="" />
+          <img className="h-[61px]" src="/logos/reseau-qet.webp" alt="" />
+          <img
```

Comment:
Le logo de Paris est avant celui de la coop carbone
```suggestion
          <img className="h-[61px]" src="/logos/ville-paris.webp" alt="" />
          <img
```